keyboards/meetlab/rena/readme.md

```diff
@@ -0,0 +1,27 @@
+# Rena - PCB
+
+![Rena](https://i.imgur.com/95rqwegh.jpg)
+
+A 59 keys keyboard with rgb.
+This keyboard use 16mhz HSE and STM32F103 as MCU.
```

Comment:
Default config is for 8MHz <https://github.com/qmk/qmk_firmware/blob/master/platforms/chibios/boards/STM32_F103_STM32DUINO/board/board.h#L34>, and you have not overridden the config.

Has this been tested on hardware?

Reply:
> Default config is for 8MHz https://github.com/qmk/qmk_firmware/blob/master/platforms/chibios/boards/STM32_F103_STM32DUINO/board/board.h#L34, and you have not overridden the config.
> 
> Has this been tested on hardware?

The 8MHZ used in the schematic, the 16MHZ used in this schematic is because of the copied STM32F401, causing such a blank loop is very sorry.